tasks/install_dispatcher.yml

```diff
@@ -0,0 +1,87 @@
+---
+# file: tasks/install_dispatcher.yml
+
+- block:
+  - name: ensure that {{ dispatcher_root }} path is ready
+    file:
+      path: '{{ dispatcher_root }}'
+      owner: '{{ web_server_user }}'
+      group: '{{ web_server_group }}'
+      state: directory
+      mode: 0755
+    notify:
+    - restart apache
+
+  - name: Create link to modules directory
+    file:
+      src: /usr/lib/apache2/modules
+      dest: /etc/apache2/modules
+      state: link
+    when: ansible_facts['os_family'] == 'Debian'
```

Comment:
Why using such conditionals instead of including related tasks for different OS versions?

Reply:
replaced